🚀 Release prep for pub app

migbro opened the pull request:
<!--Pull Request Template-->

## Description

 - Added compression to output files
 - Updated README and doc
 - Updated target release

Fixes # https://github.com/d3b-center/bixu-tracker/issues/1946

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] https://cavatica.sbgenomics.com/u/d3b-bixu/d3b-diskin-pathogenicity-dev/tasks/c4c13c91-07e7-4fc6-a073-8e0f82365d08/
- [ ] Test B

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

jharenza commented on tools/compress_annovar_output.cwl:
```diff
@@ -0,0 +1,28 @@
+    ramMin: 16000
+    coresMin: 8
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/autopvs1:v0.2.0a'
```
should this be 2.0a? https://github.com/JiguangPeng/autopvs1/tree/v2.0

migbro replied:
Created a PR here: https://github.com/d3b-center/bixtools/pull/192 - although lots of mods to the original which begs the question:
1. Do I treat it as revision to the original?
2. Give it a whole new version since it's a port?
3. Actually call it v1 since we are happy in prod and it's a modified port of the original?
Thoughts @dmiller15 ?

migbro commented:
@jharenza ok, this will be a delicate ballet. The autoPVS1 docker builds from this, and I will make it v1.0.0. However , I need to have this merged first, before the docker will build correctly. The docker pull reference in cwl has been updated, but it won't work until the PR is merged. So basically, once this is merged, then I can update the dockerfile pull (or I gues push manually) and it'll all be good!

migbro commented:
Actually, nm, to make life easier/smarter to break the dependencies loop, I will create a separate repo for our version of autoPVS1 to break this dependency loop and will update afterwards

jharenza commented on README.md:
```diff
@@ -4,11 +4,15 @@
+The major pieces of software being used are:
+ - ANNOVAR latest: The software has no versioning, but references do. See `annovar_db` section in [Recommended inputs](#recommended-inputs)
+ - InterVar v2.2.1
+ - AutoPVS1 v2.0a: Modified from the original to fit annotated KF vcf output. See [README for autoPVS1](autopvs1/README.md) for details
```
so now will be 1.0.0 (but modified from AutoPVS1 v2.0)?

migbro replied:
Correct, I will update the readme after I peel it off. It'll be cleaner and better for maintenance if needed. I'll ping again after for hopefully a final review

migbro commented:
@jharenza ok, I have updated the readme to point to the new version and location for autoPVS1. I also removed it from this repo so that it lives in one place

jharenza commented on README.md:
```diff
@@ -4,11 +4,15 @@
+The major pieces of software being used are:
+ - ANNOVAR latest: The software has no versioning, but references do. See `annovar_db` section in [Recommended inputs](#recommended-inputs)
+ - InterVar v2.2.1
+ - AutoPVS1 v1.0.1: Modified from the original to fit annotated KF vcf output. See [README for autoPVS1](https://github.com/d3b-center/D3b-autoPVS1/tree/v1.0.0#readme) for details
```
```suggestion
 - AutoPVS1 v1.0.1: Modified from AutoPVS1 v2.0 to fit annotated KF vcf output. See [README for autoPVS1](https://github.com/d3b-center/D3b-autoPVS1/tree/v1.0.0#readme) for details
```

migbro commented:
Going to consider @naqvia a yes on this PR given approval to run the wf.
